manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.locale.en-US.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: gdiObjects.xBar
+PackageVersion: 2.0.2111.26
+PackageLocale: en-US
+Publisher: gdiObjects
+PublisherUrl: https://www.gdiobjects.com/index.php
+PublisherSupportUrl: https://www.gdiobjects.com/helpdesk/
+PrivacyUrl: https://www.gdiobjects.com/privacy.php
+Author: gdiObjects
+PackageName: xBar

```

Comment:
```suggestion
PackageName: xBar 2.0
```

Reply:
> 

please keep it as xBar (to be consistent with fotoXplorer's Package Name)

Reply:
This field has to match the value in the Add/Remove Programs table shown in control panel. In my control panel, it shows as `xBar 2.0`